screwdriver/screwdriver_local.go

```diff
@@ -0,0 +1,100 @@
+package screwdriver
+
+import (
+	"fmt"
+	"net/url"
+)
+
+type localApi struct {
+	baseURL    string
+	jobName    string
+	localBuild Build
+}
+
+func NewLocal(url, jobName string, localBuild Build) (API, error) {
+	newapi := localApi{
+		url,
+		jobName,
+		localBuild,
+	}
+	return API(newapi), nil
+}
+
+func (a localApi) makeURL(path string) (*url.URL, error) {
+	version := "v4"
+	fullpath := fmt.Sprintf("%s/%s/%s", a.baseURL, version, path)
+	return url.Parse(fullpath)
+}
+
+func (a localApi) BuildFromID(buildID int) (build Build, err error) {
+	return a.localBuild, nil
+}
+
+func (a localApi) EventFromID(eventID int) (event Event, err error) {
+	event = Event{
+		0,
+		make(map[string]interface{}),
+		0,
+	}
```

Comment:
It is hard to understand what 0 is.
I think it is better to add field name.

Reply:
I fixed it.